lib/child_process.js

```diff
@@ -297,12 +297,14 @@ exports.execFile = function(file /*, args, options, callback*/) {
   return child;
 };
 
-var _deprecatedCustomFds = internalUtil.deprecate(function(options) {
-  options.stdio = options.customFds.map(function(fd) {
-    return fd === -1 ? 'pipe' : fd;
-  });
-}, 'child_process: options.customFds option is deprecated. ' +
-   'Use options.stdio instead.');
+const _deprecatedCustomFds = internalUtil.deprecate(
+  function deprecateCustomFds(options) {
+    options.stdio = options.customFds.map(function(fd) {
```

Comment:
Oh ops, sorry so maybe just give the original function the `mapCustomFds` name without using the arrow function.

I think it's the cleanest thing to do.

Sorry again for the confusion.